bokeh/_testing/util/examples.py

```diff
@@ -265,11 +237,11 @@ def store_img(self, img_data):
 
     def upload_imgs(self):
         if isfile(self.img_path):
-            trace("%s Uploading image to S3 to %s" % (green(">>>"), self.img_url_path))
-            upload_file_to_s3(self.img_path, self.img_url_path, "image/png")
+            trace("%s Uploading image to S3 to %s" % (green(">>>"), self.img_path))
+            upload_file_to_s3_by_job_id(self.img_path, "image/png")
         if isfile(self.diff_path):
-            trace("%s Uploading image to S3 to %s" % (green(">>>"), self.diff_url_path))
-            upload_file_to_s3(self.diff_path, self.diff_url_path, "image/png")
+            trace("%s Uploading image to S3 to %s" % (green(">>>"), self.diff_path))
+            upload_file_to_s3_by_job_id(self.diff_path, "image/png")
```

Comment:
I believe the paths here are the local file paths, which is why the uploads are not happening correctly

Reply:
I think the correct approach is:
```
upload_file_to_s3(self.diff_path, join(JOB_ID, self.diff_url_path), "image/png")
```
The examples' path situation is a bit messy and exacerbated by allowing for having reports with remote (uploaded) images or local images (which is useful when running tests locally).